src/cursor/cur_backup_incr.c

```diff
@@ -175,6 +176,10 @@ __curbackup_incr_next(WT_CURSOR *cursor)
              *    Return no file information as there is no new information.
              */
             if (cb->bitstring.mem == NULL || F_ISSET(cb, WT_CURBACKUP_RENAME)) {
+                cb->nbits = 0;
```

Comment:
Instead of initializing here, can we do it inside the function `__curbackup_incr_blkmod` from where the `cb` is filled?

Reply:
done 👍